### PR TITLE
enh(packaging): add rpm sign to workflow

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

add rpm sign to workflow

**Fixes** # (issue)

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software
- [ ] Updating documentation (missing information, typo...)

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

## Checklist

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have made corresponding changes to the **documentation**.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

File: .github/actions/delivery/action.yml [hunk 1]
```diff
@@ -76,7 +76,7 @@ runs:
         fi
         echo "[DEBUG] - Branch name: $BRANCHNAME"
         case "$BRANCHNAME" in
-          develop | dev-[2-9][0-9].[0-9][0-9].x)
+          develop | dev-[2-9][0-9].[0-9][0-9].x | fix-rpmsign)
```
Comment (tuntoja) [inline .github/actions/delivery/action.yml:79]: Remove this branch before merging please

File: .github/actions/delivery/action.yml [hunk 2]
```diff
@@ -98,26 +99,28 @@ runs:
...
           if [[ "${{ env.extfile }}" == "deb" ]] ; then
-            echo "[DEBUG] - https://artifactory.apps.centreon.com/artifactory/debian-$VERSION-$REPO/pool/$FILE;deb.distribution=bullseye;deb.component=main;deb.architecture=amd64"
-            curl -v -u "${{ inputs.repos_username }}":"${{ inputs.repos_password }}" -X PUT "https://artifactory.apps.centreon.com/artifactory/debian-$VERSION-$REPO/pool/$FILE;deb.distribution=bullseye;deb.component=main;deb.architecture=amd64" -T "./$FILE"
+            curl -u "${{ inputs.repo_username }}":"${{ inputs.repo_password }}" -H "Content-Type: multipart/form-data" --data-binary "@./$FILE" https://apt.centreon.com/repository/23.04-$SUBREPO/
           else
             #echo "[DEBUG] - https://artifactory.apps.centreon.com/artifactory/rpm-$VERSION-$REPO/$DISTRIB/$REPO/x86_64/centreon-collect"
             #curl -v -u "${{ inputs.repos_username }}":"${{ inputs.repos_password }}" -X PUT "https://artifactory.apps.centreon.com/artifactory/rpm-$VERSION-$REPO/$DISTRIB/$REPO/x86_64/centreon-collect/$FILE" -T "./$FILE"
-            
+
+            DISTRIB="${{ inputs.distrib }}";
```
Comment (kduret) [inline .github/actions/delivery/action.yml:116]:
```suggestion
            DISTRIB="${{ inputs.distrib }}"
```

File: .github/actions/delivery/action.yml [hunk 3]
```diff
@@ -68,6 +68,7 @@ runs:
 
     - name: Publish packages
       run: |
+        set -x
```
Comment (kduret) [inline .github/actions/delivery/action.yml:71]:
```suggestion
```

File: .github/actions/delivery/action.yml (hunk 1 above)
Comment (kduret) [reply .github/actions/delivery/action.yml:80]: Once it's removed, I'll approve it

File: .github/actions/delivery/action.yml [hunk 4]
```diff
@@ -76,7 +77,7 @@ runs:
         fi
         echo "[DEBUG] - Branch name: $BRANCHNAME"
         case "$BRANCHNAME" in
-          develop | dev-[2-9][0-9].[0-9][0-9].x)
+          develop | dev-[2-9][0-9].[0-9][0-9].x | fix-rpmsign)
```
Comment (bdauria) [inline .github/actions/delivery/action.yml:79]:
```suggestion
          develop | dev-[2-9][0-9].[0-9][0-9].x)
```

**Pull request merged**